acceptance/suites/pre_suite/foss/10_install_release_repos.rb

```diff
@@ -1,5 +1,8 @@
 step "Setup Puppet Labs Release repositories." do
   hosts.each do |host|
-    install_puppetlabs_release_repo host
+    platform = host.platform
```

Comment:
I'll take a look.  I didn't know about `confine :to, :platform => 'windows'` either, until very recently.